src/pages/atomicdex/api/v20-dev/lightning/channels/index.mdx

```diff
@@ -0,0 +1,924 @@
+
+<CodeGroup title="" tag="POST" label="METHOD_NAME">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::open_channel",
+    "params": {
+        "coin": "tBTC-lightning",
+        "node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735",
+        "amount": {
+            "type": "Max"
+        },
+        "push_msat": 0,
+        "channel_options": {
+            "proportional_fee_in_millionths_sats": 0,
+            "base_fee_msat": 1000,
+            "cltv_expiry_delta": 72,
+            "max_dust_htlc_exposure_msat": 5000000,
+            "force_close_avoidance_max_fee_satoshis": 1000
+        },
+        "channel_configs" : {
+            "counterparty_locktime": 144,
+            "our_htlc_minimum_msat": 1,
+            "negotiate_scid_privacy": false,
+            "max_inbound_in_flight_htlc_percent": 10,
+            "announced_channel": false,
+            "commit_upfront_shutdown_pubkey": true
+        }
+    },
+    "id": null
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": {
+		"uuid": "2b50e274-c173-4fa1-95f3-97f9f82ace92",
+		"node_address": "038863cf8ab91046230f561cd5b386cbff8309fa02e3f0c3ed161a3aeb64a643b9@203.132.94.196:9735"
+	},
+	"id": null
+}
+```
+</CollapsibleSection>
+
+
+## Close a Channel {{label : 'lightning::channels::close_channel', tag : 'API-v2'}}
+
+The `lightning::channels::close_channel` method closes a channel with a remote node.
+
+### Request Parameters
+
+| Parameter         | Type    | Description                                                                              |
+|-------------------|---------|------------------------------------------------------------------------------------------|
+| coin              | string  | The ticker of the coin you would like to view closed channels for.                       |
+| uuid              | string  | Then channel UUID.                                                                       |
+| force_close       | boolean | Optional, defaults to `false`. If `true`, will force closure of the channel.             |
+
+
+#### 📌 Example to force closing a channel
+
+<CodeGroup title="" tag="POST" label="close_channel">
+```json {{ mm2MethodDecorate : true }}
+{
+    "userpass": "MM2_RPC_PASSWORD",
+    "mmrpc": "2.0",
+    "method": "lightning::channels::close_channel",
+    "params": {
+        "coin": "tBTC-lightning",
+        "uuid": "2b50e274-c173-4fa1-95f3-97f9f82ace92",
+        "force_close": true
+    },
+    "id": 8
+}
+```
+</CodeGroup>
+
+
+<CollapsibleSection expandedText='Hide Response' collapsedText='Show Response'>
+
+#### Response
+
+```json
+{
+	"mmrpc": "2.0",
+	"result": "Initiated closing of channel with uuid: 2b50e274-c173-4fa1-95f3-97f9f82ace92",
```

Comment:
To see if the channel is closed or not, you can use `lightning::channel::get_channel_details`.

You can add a note for the above. Alternatively, I can add this to the response message instead of implying it, what do you think?

Reply:
I can add a note, though cant hurt to have in response

Reply:
note added under response